### DISCUSSION TITLE
ios 中 发现pagView pause 也会触发 end 回调  由于我的pagView 是复用的  我尝试去remove Listener  但是 还是会触发end回调

### DISCUSSION THREAD

**Discussion by xiaopengLOVEcode** [Tencent/libpag#1843] [General]
ios 中 发现pagView pause 也会触发 end 回调  由于我的pagView 是复用的  我尝试去remove Listener  但是 还是会触发end回![Uploading image.png…]()

**Comment by zhalice2011** [comment]
你好朋友, 你的图片上传失败了

**Comment by xiaopengLOVEcode** [comment]
您好 我补充了下图片 辛苦结合描述 看一下这个问题谢谢

发自我的iPhone


------------------ 原始邮件 ------------------
发件人: DALIZHOU ***@***.***&gt;
发送时间: 2023年11月10日 17:08
收件人: Tencent/libpag ***@***.***&gt;
抄送: xiaoxiaopeng ***@***.***&gt;, Author ***@***.***&gt;
主题: Re: [Tencent/libpag] ios 中 发现pagView pause 也会触发 end 回调  由于我的pagView 是复用的  我尝试去remove Listener  但是 还是会触发end回调 (Discussion #1843)





 
你好朋友, 你的图片上传失败了
